src/transaction.ts

```diff
@@ -1035,6 +1035,19 @@ export class Transaction implements TransactionStorageStructure {
     return new Uint8Array(encoding.encode(sTxn));
   }
 
+  attachSignature(signerAddr: string, signature: Uint8Array) {
+    const sTxn: EncodedSignedTransaction = {
```

Comment:
Perhaps verify the length of the signature? I believe it should always be 64 bytes